src/app/features/animals/components/animals-list/animals-list.component.html

```diff
@@ -1,4 +1,60 @@
-<h1>Animals</h1>
+<div class="row">
+  <div class="col">
+    <h1>Animals</h1>
+  </div>
+  <div class="col text-end">
+    <button class="btn btn-success" routerLink="/animals/add">Add new</button>
+  </div>
+</div>
 
-<!-- TODO: Add table with animals -->
-<p>Coming soon...</p>
+<ul class="nav nav-tabs mb-4">
+  <li class="nav-item">
+    <a class="nav-link" (click)="changeActiveTab('table')" [class.active]="activeTab === 'table'">Table</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" (click)="changeActiveTab('cards')" [class.active]="activeTab === 'cards'">Cards</a>
+  </li>
+</ul>
+
+<div>
+  <ng-container *ngIf="activeTab === 'table'">
+    <table class="table table-striped align-middle">
+      <thead>
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Name</th>
+        <th scope="col">Type</th>
+        <th scope="col" class="text-end">Actions</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr *ngFor="let animal of animalsService.animals; let i = index">
+        <th scope="row">{{ i + 1 }}</th>
+        <td>{{ animal.name }}</td>
+        <td>{{ animal.type }}</td>
+        <td class="text-end">
+          <button type="button" class="btn btn-danger btn-sm" (click)="removeAnimal(animal)">Remove</button>
+        </td>
+      </tr>
+      </tbody>
+    </table>
+  </ng-container>
+
+  <ng-container *ngIf="activeTab === 'cards'">
+    <div class="row">
+      <div *ngFor="let animal of animalsService.animals; let i = index" class="col-3 mb-4">
+        <div class="card" [class.border-danger]="animal.type === 'Wild'"
```

Comment:
Kartičky so zvieratami majú momentálne rôznu výšku a vyzerá to zle. Pridanie class `h-100` z Bootstrapu by malo pomôcť. Pozri [Grid cards](https://getbootstrap.com/docs/5.3/components/card/#grid-cards)

```suggestion
        <div class="card h-100" [class.border-danger]="animal.type === 'Wild'"
```